homeworks/eduardo-griboldi/contact.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact Eduardo Garcia Riboldi</title>
+</head>
+
+<body>
+   
+        <a href="file:///Users/chepegarciariboldi/Desktop/repoFrontEnd/SOLVD-FE-training/homeworks/eduardo-griboldi/home.html"> Home</a>
+        <a href="file:///Users/chepegarciariboldi/Desktop/repoFrontEnd/SOLVD-FE-training/homeworks/eduardo-griboldi/contact.html"> Contact</a>
+    <br/>
+    
+    <h1>Contact Me</h1>
+    <form>
+        <label>Full name: 
+            <input id="First-Name" type="text"  placeholder="Your name..."/> 
+            <input id="Last-Name" type="text" placeholder="Your last name..."/> 
+        </label>
+        <br />
+        <label>Phone number: 
+            <input id="Phone-number" type="text" placeholder="Your phone..." />
+        </label>
+        <br />
+        <label for="Country">Country</label>
+
+            <select name="country" id="country">
```

Comment:
here is one of the reasons why we use a convention naming haha.
`Country` its not the same as `country`